packages/components/src/DataList/DataList.stories.tsx

```diff
@@ -143,3 +144,105 @@ export const EmptyState = Template.bind({});
 EmptyState.args = {
   data: [],
 };
+
+const BreakpointTemplate: ComponentStory<typeof DataList> = args => {
+  const {
+    data,
+    /* See useCollectionQuery for example on how to load more */
+    // refresh,
+    // nextPage,
+    // loadingRefresh,
+    // loadingNextPage,
+    loadingInitialContent,
+  } = useCollectionQuery<ListQueryType>({
+    query: LIST_QUERY,
+    queryOptions: {
+      fetchPolicy: "network-only",
+      nextFetchPolicy: "cache-first",
+      client: apolloClient,
+    },
+
+    getCollectionByPath(items) {
+      return items?.allPeople;
+    },
+  });
+
+  /* For when we start creating a loading state */
+  // const { loading } = getLoadingState(
+  //   loadingInitialContent,
+  //   loadingRefresh,
+  //   loadingNextPage,
+  // );
+
+  const items = data?.allPeople.edges || [];
+
+  const randomTags = ["SW", "commercial", "pets", "fence"];
+  const mappedData = items.map(({ node }) => ({
+    id: node.id,
+    label: node.name,
+    home: node.homeworld.name,
+    tags: [
+      node.gender,
+      ...(node.hairColor?.split(", ") || []),
+      ...(node.skinColor?.split(", ") || []),
+      ...randomTags.slice(
+        Math.round(Math.random() * 2),
+        Math.round(3 + Math.random() * 4),
+      ),
+    ].filter(t => t !== "n/a"),
+    homePopulation: node.homeworld.population?.toLocaleString(),
+    created: new Date(node.created),
+  }));
+
+  return (
+    <DataList
+      loading={loadingInitialContent}
+      totalCount={null}
+      {...args}
+      data={(args.data as typeof mappedData) || mappedData}
+      headers={{
+        label: "Name",
+        home: "Home world",
+        tags: "Attributes",
+        homePopulation: "Home world population",
+        created: "Created",
+      }}
+    >
+      <DataList.Layout size="lg">
+        {(item: DataListItemType<typeof mappedData>) => (
+          <Grid alignItems="center">
+            <Grid.Cell size={{ xs: 5 }}>
+              <Grid alignItems="center">
+                <Grid.Cell size={{ xs: 6 }}>{item.label}</Grid.Cell>
+                <Grid.Cell size={{ xs: 6 }}>{item.home}</Grid.Cell>
+              </Grid>
+            </Grid.Cell>
+            <Grid.Cell size={{ xs: 2 }}>{item.homePopulation}</Grid.Cell>
+            <Grid.Cell size={{ xs: 3 }}>{item.tags}</Grid.Cell>
+            <Grid.Cell size={{ xs: 2 }}>{item.created}</Grid.Cell>
+          </Grid>
+        )}
+      </DataList.Layout>
+      <DataList.Layout size={"xs"} showHeader={false}>
+        {(item: DataListItemType<typeof mappedData>) => (
+          <Card>
```

Comment:
I like this story a lot - but I'm wondering if we should be putting them into cards.  The figma only has the divider on the mobile web views without an extra box around the line item.  Should we mimic that?